### ISSUE TITLE
Global Font Size Brand Standard

### ISSUE THREAD

**Issue by sarah0arnold** [aemsites/clarkcountynv#1351]
**Component block:** None, global

**Design:** https://www.figma.com/design/T2bgro0OiuLqxvm1uLhJZ2/Clark-County-NV?node-id=132-1003&t=7ZXrRUQI9M74s5dW-0

As a site visitor, I want consistent font sizes across the website that are accessible so that I can easily find the content I need

**Requirements:** 
Global body text font sizes and styles should be updated to brand guidelines: 
Body M PP Neue Montreal, 18px, book
Body S PP Neue Montreal, 16px, book
Body XS PP Neue Montreal, 14px, book
All component blocks coded to leverage the above styles should display the corresponding font, font size, font weight, and style

**Acceptance Criteria:** 
1. Given I am viewing a component with medium body font, when the page renders, then the text displays as PP Neue Montreal, 18px at book weight
2. Given I am viewing a component with small body font, when the page renders, then the text displays as PP Neue Montreal, 16px at book weight
3. Given I am viewing a component with extra small body font, when the page renders, then the text displays as PP Neue Montreal, 14px at book weight

**Comment by beccasuchanek** [comment]
@aaronstewart-dev This ticket passed all ACs, see details below:

**Test Page URL:** https://develop--clarkcountynv--aemsites.aem.page/drafts/bsuchanek/tickets/global-font-size-brand-standard

1. ✅ Given I am viewing a component with medium body font, when the page renders, then the text displays as PP Neue Montreal, 18px at book weight

<img width="1547" height="779" alt="Image" src="https://github.com/user-attachments/assets/05a579d5-4e1f-434f-a2a9-632c713707db" />

2. ⚠️ Given I am viewing a component with small body font, when the page renders, then the text displays as PP Neue Montreal, 16px at book weight **This cannot be tested due to no use case within the site and component but this is implemented in code and will need to be test when use cases are developed.** @sarah0arnold 
3. ⚠️ Given I am viewing a component with extra small body font, when the page renders, then the text displays as PP Neue Montreal, 14px at book weight **This cannot be tested due to no use case within the site and component but this is implemented in code and will need to be test when use cases are developed.** @sarah0arnold

**Comment by sarah0arnold** [comment]
Font included as part of all blocks reviewed in UAT session with project team 8/4/25 and passes with no feedback. Closing ticket
